pkg/root/signing_config_test.go

```diff
@@ -79,7 +79,7 @@ func TestSelectService(t *testing.T) {
 			URL:                 "url_no_end",
 			MajorAPIVersion:     2,
 			ValidityPeriodStart: farFuture,
-			ValidityPeriodEnd:   time.Time{},
+			ValidityPeriodEnd:   time.Unix(0, 0),
```

Comment:
this is actually quite surprising and we'll have to think this through: clients currently have to consider 1970-01-01 as a magic number that means the value was not set.

ugh, I wish we didn't use a wire format as a file format :(

Reply:
as expected I really how this makes the code look (suddenly we're dealing with pointers everywhere). But I guess this is what properly dealing with protobufs means... I'll include the fix as an independent commit, not sure if it makes sense